server/protocols/jmap-rfc8621-integration-tests/distributed-jmap-rfc8621-integration-tests/src/test/resources/imapserver.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+<imapservers>
+    <imapserver enabled="true">
+        <jmxName>imapserver</jmxName>
```

Comment:
are we using imap at all in this project?
on two different ports?

Reply:
I dont know yet, i just copy those files when it needed.I will try to understand it.

Reply:
> are we using imap at all in this project?
> on two different ports?

Yes, because long ago we decided to always start protocols, event the ones we don't use.

(an minor test runtime optimization I proposed was rejected because of this)

Now that is said, my position is unchanged: I would agree to disable protocols we don't use ;-)

Reply:
I tried in my computer with:
- imapserver.xml
- lmtpserver.xml
- pop3server.xml
- smtpserver/xml
without content and is breaking nothing. So it not needed, would you agree with it?